winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,149 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the customization features such as BorderThickness, BorderBrush, Background, FontSize and GradientBackground in WinUI AvatarView control here.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Customization in WinUI AvatarView 
\ No newline at end of file
+# Customization in WinUI AvatarView 
+
+The `AvatarView` control offers a range of customization options, including the ability to modify its colors, background, and more. You can utilize the following properties to customize this control:
+
+## Border
+
+Personalize the border of `AvatarView` control by adjusting the border brush and border thickness properties to achieve the desired look.
```

Comment:
@MadhumithanJayapathi Customize the border of `AvatarView` control by using the BorderBrush and BorderThickness

Reply:
@Manivannan-E Changed the Content of Border as mentioned.